web/src/components/RunDetailAutomate/RunDetailAutomate.tsx

```diff
@@ -1,23 +1,29 @@
+import {snakeCase} from 'lodash';
 import Test from 'models/Test.model';
+import {useState} from 'react';
 import TestRun from 'models/TestRun.model';
 import * as S from './RunDetailAutomate.styled';
-import TestDefinition from '../RunDetailAutomateDefinition';
+import RunDetailAutomateDefinition from '../RunDetailAutomateDefinition';
 import RunDetailAutomateMethods from '../RunDetailAutomateMethods/RunDetailAutomateMethods';
 
 interface IProps {
   test: Test;
   run: TestRun;
 }
 
-const RunDetailAutomate = ({test, run}: IProps) => (
-  <S.Container>
-    <S.SectionLeft>
-      <TestDefinition test={test} />
-    </S.SectionLeft>
-    <S.SectionRight>
-      <RunDetailAutomateMethods test={test} run={run} />
-    </S.SectionRight>
-  </S.Container>
-);
+const RunDetailAutomate = ({test, run}: IProps) => {
+  const [fileName, setFileName] = useState<string>(`${snakeCase(test.name)}.yaml`);
```

Comment:
I went back and forth on having this to be a local state or something from a provider, but to keep it simple I just add it as a state. If we need to carry over multiple things for the techniques we can have an automate provider instead.

Reply:
I like it. Keep it simple.